src/sentry/rules/processing/delayed_processing.py

```diff
@@ -240,10 +238,13 @@ def get_group_to_groupevent(
 @redis_buffer_registry.add_handler(BufferHookEvent.FLUSH)
 def process_delayed_alert_conditions(buffer: RedisBuffer) -> None:
     with metrics.timer("delayed_processing.process_all_conditions.duration"):
+        fetch_time = datetime.now(tz=timezone.utc)
         project_ids = buffer.get_set(PROJECT_ID_BUFFER_LIST_KEY)
```

Comment:
I just realised I've been confused by this name the whole time. I thought it means `get then set`, but it means`get the values from a set`.

It might make sense to rename it as part of the changes here, `get_range_from_set` or something?

Reply:
Hahaha oh no 😅 yeah I think I had like `get_list` before and then it changed to a set ... what about `get_sorted_set` since we also have `get_hash` and `push_to_sorted_set`?

Reply:
Yeah that seems fine, let's go with that